src/main/java/org/example/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,21 @@
 package org.example.bookstore.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
```

Comment:
Just in case, you can add an informative message for users, but probably it is not necessary

Reply:
I agree, better to add specific error message